python/tchannel/tornado/tchannel.py

```diff
@@ -86,6 +87,10 @@ def __init__(self, hostport=None, process_name=None):
         self.event_emitter = EventEmitter()
         self.hooks = EventRegistrar(self.event_emitter)
 
+        if known_peers:
```

Comment:
.peers is part of the TChannel interface. Accessing it and adding/getting peers is completely okay.